src/Components/Filtering/RangeFilter.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable no-unused-vars */
```

Comment:
This file i sone that I think could be reorganized when merging with search. It is a little odd that the multiselects are declared here rather than having the two different filter types completely separated and displayed in a larger search/filter card component. I think that as you merge with search the reorg will make sense, but for now, it would be nice if you could refactor this and think about which component structure best suits your purposes.